src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Object/ObjectConverterFactory.cs

```diff
@@ -99,6 +101,7 @@ public override JsonConverter CreateConverter(Type typeToConvert, JsonSerializer
             return converter;
         }
 
+        [UnconditionalSuppressMessage("AOT", "IL3050:Calling members annotated with 'RequiresDynamicCodeAttribute' may break functionality when AOT compiling.", Justification = "<Pending>")]
```

Comment:
Unless the key type and value type are reference types or code for that instantiation is available through other means, this is not safe to suppress.